src/picongpu/include/simulationControl/MovingWindow.hpp

```diff
@@ -105,12 +105,14 @@ class MovingWindow
                 cellSizeInMoveDirection;
             /* Is the time step when the virtual particle **passed** the moving window
              * in the current to the next step
+         * Signed type of firstMoveStep to allow for edge case slide_point = 0.0
```

Comment:
Ok, the `sed`-command suggested by `continuous-integration/travis-ci/pr` did not work as expected. I fixed it manually.